tests/utils/exec/rosacli/ocm_resource_service.go

```diff
@@ -55,6 +55,7 @@ type OCMResourceService interface {
 
 	ListOIDCConfig() (OIDCConfigList, bytes.Buffer, error)
 	ListInstanceTypes() (InstanceTypesList, bytes.Buffer, error)
+	ListInstanceTypesByRegion(flags ...string) (InstanceTypesList, bytes.Buffer, error)
```

Comment:
If you name your method `ListInstanceTypesByRegion`, then the first parameter of the method should be `region`
An alternative would be to remove that new method and  directly update `ListInstanceTypes()` by adding the `flags ...string` parameter. This should not impact other calls.

Reply:
Fixed